index.html

```diff
@@ -6,20 +6,49 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="style.css">
   <link rel="stylesheet" href="queries.css">
+  <script src="menu.js"></script>
```

Comment:
- Fix your icons in your ``menu`` once that you click on your ``hamburger menu``, the ``X`` keeps displayed.

You can work displaying and hidding the icons; using the``class`` icons in a example.